### DISCUSSION TITLE
Internet connection check as3

### DISCUSSION THREAD

**Discussion by Schicksal44** [community/community#141311] [Programming Help] [answered]
### Body

How to implement internet connection check as3?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by oleksii-dmytriiev-ua** [comment, marked as answer]
Try this:

- 	  var url : String = 'https://www.google.com/';
		  var request : URLRequest = new URLRequest  ( url );
		  var loader : URLLoader = new URLLoader ();
		  loader.addEventListener(IOErrorEvent.IO_ERROR, 
			  function ioErrorHandler(event:IOErrorEvent):void 
			  {
				  trace("ioErrorHandler: " + event);
			  }
		  );
		  
		  loader.addEventListener( Event.COMPLETE,
			  function( e:Event ) : void
			  {
				  OStatisticSystem.clickMoregamesButton();
				  Partner.iAdManager.ShowAdMoreGames();
			  }
		  );		
		  loader.load ( request );

**Reply by Schicksal44** [reply]
Thank you
